### PR TITLE
[OPS-6684] Bake a new redis.

### PR DESCRIPTION
**Pull request by cafuego**
Available as unocha/redis:5.0.7-202006-01

### CODE REVIEW

File: alpine-base-redis/Dockerfile.tmpl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM unocha/alpine-base-s6:%%UPSTREAM%%
 
-MAINTAINER "Serban TEODORESCU <teodorescu.serban@gmail.com>"
+MAINTAINER UN-OCHA Operations <ops+docker@humanitarianresponse.info>
```
Comment (vauxia) [inline alpine-base-redis/Dockerfile.tmpl:3]: I'm so confused about humanitarianresponse.info vs reliefweb.int.

**Review by vauxia — APPROVED** [review]
All seems very reasonable.

File: alpine-base-redis/Dockerfile.tmpl (hunk 1 above)
Comment (cafuego) [reply alpine-base-redis/Dockerfile.tmpl:3]: Legacy vs current. This is just a copy & paste job. Doesn't matter that much anyway, we never look at it or use it for anything ;-)

Comment (teodorescuserban) [reply alpine-base-redis/Dockerfile.tmpl:3]: I think `MAINTAINER` is deprecated.

Comment (teodorescuserban) [reply alpine-base-redis/Dockerfile.tmpl:3]: https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

**Review by teodorescuserban — APPROVED** [review]
removing maintainer is good.
removing _that_ maintainer is even better :)

**Pull request merged**